readme.md

```diff
@@ -1404,9 +1404,11 @@ in a `next.config.js` file like so:
 
 ```js
 module.exports = {
+  // Make sure to include 'mdx' files as pages
+  pageExtensions: ['mdx', 'tsx', 'ts', 'jsx', 'js'],
```

Comment:
This is not required I believe. But it is often useful.
I think it makes sense to split the two things going on here.

This is needed if you want `.mdx` files to show up as whole pages. Right?

The other is so that `.mdx` files can be imported: `import Content from "./somefile.mdx"`.

How about this?

```suggestion
  // Support MDX files as pages:
  pageExtensions: ['mdx', 'tsx', 'ts', 'jsx', 'js'],
```

Reply:
Wonderful idea, thank you!